1.28.2/rockcraft.yaml

```diff
@@ -49,7 +49,12 @@ parts:
       chmod +x /usr/local/bin/bazel
 
       # setup required libs
-      wget -q https://github.com/llvm/llvm-project/releases/download/llvmorg-18.1.8/clang+llvm-18.1.8-x86_64-linux-gnu-ubuntu-18.04.tar.xz
+      url="https://github.com/llvm/llvm-project/releases/download/llvmorg-18.1.8/clang+llvm-18.1.8-x86_64-linux-gnu-ubuntu-18.04.tar.xz"
+      if [ "$(uname -m)" != "x86_64" ]; then
```

Comment:
let's not make assumptions here.
Explicitly check for `arm` and `amd` architectures and raise a proper error if neither of them is returned by `uname -m` (e.g. there may be issues fetching the arch and the current implementation would resolve this to `arm`